src/style.css

```diff
@@ -851,24 +849,32 @@ blockquote p:after {
   font-weight: 100;
 }
 
-.rule-content h1, .rule-content h2, .rule-content h3, .rule-content h4, .rule-content h5, .rule-content h6, .rule-content .h1, .rule-content .h2, .rule-content .h3, .rule-content .h4, .rule-content .h5, .rule-content .h6 {
-  margin-top: 2rem;
+.rule-content h1, .rule-content .h1 {
+  font-size: 2.5rem;
 }
-
-.rule-content h1 {
-  color: #cc4141;
+.rule-content h2, .rule-content .h2 {
+  font-size: 2rem;
 }
-
-.rule-content h3 {
-  color: #cc4141;
+.rule-content h3, .rule-content .h3 {
+  font-size: 1.75rem;
 }
-
-.rule-content h4 {
-  color: #414141;
+.rule-content h4, .rule-content .h4 {
+  font-size: 1.5rem;
+}
+.rule-content h5, .rule-content .h5 {
+  font-size: 1.25rem;
+}
+.rule-content h6, .rule-content .h6 {
+  font-size: 1rem;
 }
+  
+.rule-content h1, .rule-content h2, .rule-content h3, .rule-content h4, .rule-content h5, .rule-content h6, .rule-content .h1, .rule-content .h2, .rule-content .h3, .rule-content .h4, .rule-content .h5, .rule-content .h6 {
+  margin-top: 2rem;
+}
+  .rule-header-container h1 {margin-top: 1rem;}
 
-.rule-content h4 {
-  word-break: break-word;
+.rule-content h1, .rule-content h2, .rule-content h3 {
```

Comment:
@tiagov8 As per our previous conversations, could we also set text color #000 for h4-h6 (vs body #333) this would help further differentiate h6 from paragraph text.

Reply:
Done - line 880